tests/functional/cnv/test_vms.py

```diff
@@ -0,0 +1,144 @@
+import logging
+import pytest
+import subprocess
+import os
+import hashlib
+from ocs_ci.framework.testlib import E2ETest
+from ocs_ci.framework.pytest_customization.marks import magenta_squad
+from ocs_ci.helpers.cnv_helpers import cal_md5sum_vm
+from ocs_ci.helpers.performance_lib import run_oc_command
+from ocs_ci.helpers.keyrotation_helper import PVKeyrotation
+from ocs_ci.utility.utils import run_cmd
+
+logger = logging.getLogger(__name__)
+
+
+class TestCNVVM(E2ETest):
+    """
+    Includes tests related to CNV workloads on MDR environment.
+
+    """
+
+    @pytest.fixture(autouse=True)
+    def setup(self, project_factory, multi_cnv_workload):
+
```

Comment:
please add doc strings for this function

Reply:
Done